### ISSUE TITLE
[INFO] gewu. Step: 20000. Time Elapsed: 60.900 s. Mean Reward: -181.860. Std of Reward: 79.397. Training. 已中止 (核心已转储)

### ISSUE THREAD

**Issue by love1994love** [loongOpen/Unity-RL-Playground#7]
在第三步，训练机器人时，运行mlagents-learn trainer_config.yaml --run-id=tinker --force开始训练，出现以下错误：
[INFO] Listening on port 5004. Start training by pressing the Play button in the Unity Editor.
[INFO] Connected to Unity environment with package version 2.3.0-exp.3 and communication version 1.5.0
[INFO] Connected new brain: gewu?team=0
[WARNING] Deleting TensorBoard data events.out.tfevents.1742807617.u2204.600086.0.meta that was left over from a previous run.
[WARNING] Deleting TensorBoard data events.out.tfevents.1742807617.u2204.600086.0 that was left over from a previous run.
[INFO] Hyperparameters for behavior name gewu: 
	trainer_type:	ppo
	hyperparameters:	
	  batch_size:	2048
	  buffer_size:	20480
	  learning_rate:	0.0003
	  beta:	0.005
	  epsilon:	0.2
	  lambd:	0.95
	  num_epoch:	3
	  learning_rate_schedule:	linear
	  beta_schedule:	linear
	  epsilon_schedule:	linear
	network_settings:	
	  normalize:	True
	  hidden_units:	512
	  num_layers:	3
	  vis_encode_type:	simple
	  memory:	None
	  goal_conditioning_type:	hyper
	  deterministic:	False
	reward_signals:	
	  extrinsic:	
	    gamma:	0.995
	    strength:	1.0
	    network_settings:	
	      normalize:	False
	      hidden_units:	128
	      num_layers:	2
	      vis_encode_type:	simple
	      memory:	None
	      goal_conditioning_type:	hyper
	      deterministic:	False
	init_path:	None
	keep_checkpoints:	5
	checkpoint_interval:	400000
	max_steps:	20000000
	time_horizon:	1000
	summary_freq:	20000
	threaded:	False
	self_play:	None
	behavioral_cloning:	None
[INFO] gewu. Step: 20000. Time Elapsed: 60.900 s. Mean Reward: -181.860. Std of Reward: 79.397. Training.
已中止 (核心已转储)

**Comment by linqi-ye** [comment]
出现[INFO] gewu. Step: 20000……说明成功开始训练了，可试试重启软件训练，或试试训练其他机器人

**Comment by linqi-ye** [comment]
pip3 install torch==1.7.1+cpu -f https://download.pytorch.org/whl/torch_stable.html
运行上面语句重装pytorch可解决问题
